builtin/check-ref-format.c

```diff
@@ -53,31 +56,28 @@ static int check_ref_format_branch(const char *arg)
 
 int cmd_check_ref_format(int argc, const char **argv, const char *prefix)
 {
-	int i;
+	enum {
+		CHECK_REF_FORMAT_BRANCH,
+	};
```

Comment:
You will want to
- declare a variable of that type, not only an `enum`, and use that in the `OPT_CMDMODE` construct below
- have another command-mode, e.g. `CHECK_REFNAME_FORMAT`, and assign the default mode.

I.e. something like this:

```c
enum {
    CHECK_REFNAME_FORMAT = 0,
    CHECK_REF_FORMAT_BRANCH,
} mode = CHECK_REFNAME_FORMAT;
[...]
OPT_CMDMODE(0 , "branch", &mode, N_("check for valid branch name"), CHECK_REF_FORMAT_BRANCH),
```